num1: 10.0
num2: 5.0
op: +